src/mjolnir/ingest_transit.cc

```diff
@@ -234,7 +234,7 @@ std::priority_queue<tileTransitInfo> select_transit_tiles(const boost::property_
 void setup_stops(Transit& tile,
                  const gtfs::Stop& tile_stop,
                  GraphId& node_id,
-                 std::unordered_map<feedObject, GraphId>& stop_graphIds,
+                 std::unordered_map<feedObject, GraphId>& platform_node_ids,
```

Comment:
when we write the stops (nodes) into a transit pbf tile we keep track of the gtfs ids assigned to those node which are platforms, thats what this map does. we need that because later we will ingest the part of the feed that references those platforms to connected them with routes/trips at some sort of schedule.